crt_portal/cts_forms/templates/forms/report_primary_complaint.html

```diff
@@ -31,4 +31,7 @@
 {% block page_js %}
   {{ block.super }}
   <script src="{% static 'js/primary_complaint.js' %}"></script>
+  <script src="https://touchpoints.app.cloud.gov/touchpoints/e0517981/js" async></script>
+  <script src="{% static 'js/modal.js' %}"></script>
+  <script src="{% static 'js/redirect-modal.js' %}"></script>
```

Comment:
We don't need the `js/modal.js` or the `js/redirect-modal.js` included here. These are unrelated to touchpoints.

Reply:
@cantsin Removed all modal js files. Please review and merge.